modules/administration/pages/virtualization.adoc

```diff
@@ -463,6 +463,8 @@ Refer to <<sec.virtualization.vmguest-manage.status>> for details.
 
 You can also perform more advanced {vmguest} editing tasks on Salt minions, such as adding or removing disks and network interfaces, and changing the display type.
 
+
+
```

Comment:
Are these blank lines needed in this commit?

Reply:
Technically, no, but for readability we like to have three lines of whitespace between sections.